app/scripts/polling_location_finder.js

```diff
@@ -78,8 +86,101 @@ define(['jquery', 'geojson', 'json!vendor/ELECTIONS_WardsPrecincts.geojson', 'js
         return encodeURI(url + destination);
     }
 
-    return function(latLng, successCallback, errorCallback) {
+    function getHours(place) {
+
+        var hours = [];
+        moment.tz.add('America/New_York|EST EDT|50 40|0101|1Lz50 1zb0 Op0');
+        
+        jQuery.each(place, function(index, timeInterval) {
+            var range = moment.range(timeInterval);
```

Comment:
Hmmm I think you're right about this! It wasn't originally working but that might've been for other reasons